### ISSUE TITLE
Replace platform tip with host tip (from which you'll get a cut anyway)

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#6398]
Platform tips are misleading and not fair. 
Either open collective is a free open source platform and you'd be then totally entitled to ask people for a platform tip, either it's a software platform that hosts have to pay to use (like a zoom account). Right now, it's both. But you can't have your cake and and eat it too. It's a non issue for the hosts that you are directly managing (since in that case it doesn't make any difference). But it's a real issue for 3rd party hosts.

<img width="1005" alt="Screenshot 2023-02-08 at 10 25 11 AM" src="https://user-images.githubusercontent.com/74358/217503411-076538b5-2ad7-465b-952c-320fed0950c9.png">

The marginal cost for any new donation is 0 for the platform. But it's non zero for the host that will have to manually process expenses submitted to access that money donated.

Imagine a coffee shop where the tip box would be for the coffee machine, not for the person using the machine to make your coffee.

That's why people think that they are tipping the host, because that's the primary human connection they have.

**Comment by xdamman** [comment]
To be clear, this is not about reducing the overall income for Open Collective (such change wouldn't make any difference for the largest hosts directly managed by you), it's about not misleading users and independent hosts. If needed, increase the cut for those hosts and/or set a monthly subscription fee.

To put things in perspective, Open Collective is making €1455/month out of the allforclimate community, a community of volunteers working on climate justice, without contributing to the salary of anyone managing that host.

<img width="836" alt="Screenshot 2023-02-08 at 10 42 54 AM" src="https://user-images.githubusercontent.com/74358/217507501-1ec146c1-4520-4af9-ad6f-ee7576d0b182.png">

I feel like a fairer deal would be to consider those platform tips as tips for the host and include that in your calculation of your fees of 15% (or whatever new percentage you'd need to make the math work for you). Right now, in this latest bill, Open Collective is effectively eating 53% of all the fees and tips.

**Comment by gusaus** [comment]
Seems like the problems and solutions in https://github.com/opencollective/opencollective/issues/6377 might be related?

**Comment by alanna** [comment]
If you can't cover host operating costs with current host fees, you need to increase your host fee. You should set it at a level actually sustainable for the work the host must do, or seek outside funding to makeup the gap. Open Collective has to get income from hosts somehow, and that is even more important for the ones who charge low or no fees, since the platform fee take is very low in those cases, so we make up for it in optional tips.

**Comment by gusaus** [comment]
Oh right - so self-hosted collectives can set their own host fees? 

Ideally, they'd also be able to set defaults (#6252 is related) or hide the platform tip (for a fee). Isn't at least the latter what's being proposed in #6377?

**Comment by alanna** [comment]
Self-hosted / Independent Collectives have no Host Fees. No they cannot hide or opt out of tips, as that's the only way Open Collective have revenue from them.

**Comment by gusaus** [comment]
Ok - guess I misunderstood https://github.com/opencollective/opencollective/issues/6398#issuecomment-1429068694 

With regards to hiding tips (for a platform fee), isn't that what @Betree and some customers were proposing in https://github.com/opencollective/opencollective/issues/6377?

**Comment by xdamman** [comment]
@alanna this issue is not about how much the host is getting (and how much they should charge), the issue is about the interface misleading the end user. When you give a tip, you expect it to go to the operator of the machine, not to the vendor of the machine.

So tips should go to the host, then you can charge whatever you want to the host to make the math work for you. But there is a clear billing relationship between the host and you, and between the users and their host.

The thing is that in practice, those tips do go directly to the host when the end user makes a wire transfer (adding to the confusion where they think the tip is for the host).

If you want to offer a free tier to 3rd party hosts at the condition that 100% of the host tip goes to you, that's fine.
If you charge a monthly fee to the host, then arguably the commission should be lower (see for example [shopify's pricing](https://shopify.com/pricing)).

Another way to look at it: offer hosts –as a paid option– the ability to remove "platform tips".

(thanks @gusaus for making all those links with other issues / comments)

**Comment by znarf** [comment]
> this issue is not about how much the host is getting (and how much they should charge)

It's slightly about it because it seems you want Host Tips as extra revenue to sustain the host operations.

> Another way to look at it: offer hosts –as a paid option– the ability to remove "platform tips".

For Fiscal Hosts that don't like Platform Tips, it's always on the table. We just to have to find a model that give Open Collective (OC) enough revenue to justify having the Fiscal Host on the platform and keep supporting them.

For All For Climate (A4C), OC made in 2022 around $9K in platform tip, and ~$3K in revenue share (15% of host fees). It's respectively 1.11% and 0.42% of total money raised on A4C collectives. In the meantime A4C made around ~$20K in net host fees, 2.47% of total money raised.

To be honest, I think 1.5% of total money raised is already on the low side as income for the platform. But as a math exercise, we could start from here and ask the question: looking at A4C numbers, how could we keep $12K or 1.5% in annual revenue for OC without relying on Platform Tips?

**Comment by alanna** [comment]
Tips were designed to be a compromise that allowed flexibility for hosts that charge low or no fees to still have a way to support the platform without putting financial pressure on hosts without a huge budget to spend. The alternative would be something like enforcing a minimum 10% host fee or something so the math works when OC takes a % of that as platform fees (and no tips). But we have hosts who don't want to charge higher fees to Collectives and we want to give them flexibility.

**Comment by gusaus** [comment]
The problem is the platform simply doesn't work for several of your customers (the collectives). Different pricing and functionality depending on the host (for example OSC https://github.com/opencollective/opencollective/issues/6252#issuecomment-1381622473) only compounds that. 

For example, most of the Drupal collectives [having any type of success](https://discover.opencollective.com/?tag=drupal) are Drupal Camps/events (largely due to [previous support](https://www.drupal.org/association/blog/a-more-sustainable-framework-for-fiscal-sponsorship-of-drupal-camps)). Currently, they are using a mix of self-hosted, OSC, and OCF, but more and more are leaning towards OSC because a tip on event tickets and sponsorships is confusing (even though OCF may provide more benefits as a fiscal host).

With more modules/addons using Open Collective is essentially a payment gateway, the problem is solved via integration (for use cases where the tip makes no sense) but not for those discovering and purchasing through the platform (for certain hosts).

Curious why there hasn't been further progress/discussion on providing the ability to hide tips (on the collective admin/tier level) for a fee https://github.com/opencollective/opencollective/issues/6377#issuecomment-1414484659. Seems like what @xdamman is proposing could be a compliment to that?

FWIW after additional discussion in some of the largest open source communities on open collective, I'll reiterate that 'not' providing the tip option for OSC collectives is odd (and a missed opportunity). Supporters of open source projects are the ones that get Open Collective is open source and would gladly tip. They're also interested/able to support and contribute to the platform directly (if encouraged and enabled). 

Pretty sure there will be more revenue (via existing/new collectives and direct support/contributions) if the pricing and functionality were constant across all hosts.

**Comment by alanna** [comment]
I completely agree that pricing is currently messy and confusing, and we would all be served by a clearer model. It's just really hard to make sorting it out the priority when such a small percentage of revenue comes from hosts other than partner hosts (OSC/OCF/OCE, which have a 50% revenue share with OC inc), or from tips. It's not perfect but it's working well enough. Right now [OC's strategy](https://blog.opencollective.com/oc-strategy-2023-2024/) is to focus on the needs of these large hosts and we have to keep focused on that strategy. We can't take everything on at once and there are approximately eighty zillion other important problems to solve. Fixing pricing will be a fairly complex and hairy process crossing strategy, business, finance, and tech, and we likely won't be able to properly work on it until the strategy phase after this current one, which I very much hope and believe will be focused on making OC work for all the other hosts besides those three run by OC, and scaling up the network majorly. But it's not quite time yet. That's still my dream for OC, a big network of diverse hosts, most of which we don't run, and a platform and pricing model that supports that. That's the scalable dream. But we're not quite there.

**Comment by gusaus** [comment]
I appreciate that and understand changes in pricing would be complicated. The only reason I'm mentioning is my assumption has been that OSC wouldn't 'include' the ability to tip without making their pricing consistent with OCF/OCE (which are similar, right?). But now reading back through some of the earlier/related discussions https://github.com/opencollective/opencollective/issues/6252#issuecomment-1381622473, it seemed like something @BenJam was open to future exploration.

>The philosophy applied here is that we don't accept tips on transactions that Open Collective Inc. already benefits from. We may experiment with switching to platform tips in the future but, for the moment at least, we're happy supporting Open Collective Inc. as we are.  

Wondering if OSC could potentially enable platform tips while keeping the pricing the same.

The issue isn't the pricing as much as consistancy in terms of platform behaviour and control. Again, it looked like https://github.com/opencollective/opencollective/issues/6377 would be big step in terms of solving the problem. Is there a tecnical issue that would prevent that from being implimented?

**Comment by xdamman** [comment]
> FWIW after additional discussion in some of the largest open source communities on open collective, I'll reiterate that 'not' providing the tip option for OSC collectives is odd (and a missed opportunity). Supporters of open source projects are the ones that get Open Collective is open source and would gladly tip.

Oh wow, I had no idea this was going on. Indeed, of all communities, this is the one that would understand the platform tip to fund the open source software that open collective is developing under MIT license. Also since the host has been created and is still managed by OC, there is no confusion here on where the tip is going to.


@znarf thank you for computing those numbers. They are quite misleading though as they include all the donations made manually ("add funds"), for which the platform doesn't automate anything. For those, the platform behaves like a regular accounting software, whose license model is never based on a percentage of the amount of money under management. Could you re-run those numbers with only credit card donations?

I think it could be interesting to offer a license model, e.g. €5/collective/month. A4C has about 200 collectives, that would be €1000 a month or €12k a year (maybe make it a tiered model, e.g. up to 100 collectives, up to 2 or 300, up to 500, up to 1000, etc.).

Again this is not about how much money OC or the host is getting, it's about avoiding misleading the end user and therefore creating confusion and frustration. I'm sure we can all agree that is not the goal. I'm just reporting feedback from the ground, from the climate justice community of allforclimate (it's only one perspective, I'm sure other communities, especially the open source community, and the US community used to tipping, might have a very different point of view).

**Comment by gusaus** [comment]
> this is not about how much money OC or the host is getting, it's about avoiding misleading the end user and therefore creating confusion and frustration.

Obviously, it's all intertwined. More money flowing through collectives is good for everybody and right now several customers (your collectives) are having a hard time sustaining due to what should be 'solvable' limitations with the platform...especially considering the platform is open source.

While our OCF collectives have technical capacity to solve customer-facing problems through integration and third-party platforms (now that our url parameter workaround https://github.com/opencollective/opencollective/issues/6252#issuecomment-1379493216 was taken away), we're short on time and funds to do the extra work. But without the [ability to opt-out of platform tips in some cases](https://github.com/opencollective/opencollective/issues/6377) (for a fee), we still run the risk of misleading supporters finding us through https://discover.opencollective.com/

It's been almost two months since @CaptainStack posted https://github.com/opencollective/opencollective/issues/6252. Pointing out here as it spurred on many of the same discussions about problems and solutions. One of which I'd like to revisit, update, and move forward https://github.com/opencollective/opencollective/issues/6252#issuecomment-1385812818. 

Considering our [community contributor group](https://opencollective.com/community-contributor-group) has grown in that time, the only barrier to solving these problems together are funds. That too should be a solvable problem if we're proactive about raising funds for https://opencollective.com/opencollective (assuming that's the collective associated with the open source https://github.com/opencollective/opencollective).

**Comment by znarf** [comment]
> I think it could be interesting to offer a license model, e.g. €5/collective/month. A4C has about 200 collectives, that would be €1000 a month or €12k a year (maybe make it a tiered model, e.g. up to 100 collectives, up to 2 or 300, up to 500, up to 1000, etc.)

@xdamman If A4C wants to pay €5/collective/month on a monthly bill, then I think Open Collective could deactivate platform tips for the host, make sense for me. Should we try to move forward with that proposal?

**Comment by gusaus** [comment]
> Open Collective could deactivate platform tips for the host

So would all self-hosted collectives have this option? What about OCF hosted collectives?

Again, all the affected collectives I'm aware of 'want' the tip as an option. They just need the ability to hide on specific tiers/use cases where the default tip could be perceived as expensive, confusing, and/or misleading.

What @Betree was proposing in #6377 would resolve the problems for your customers. The fixed platform fee (for hiding on specific tiers) + additional/increased tips and contributions (especially if OSC enables tips) should generate more (not less) revenue for the platform and team.  

> For the (2) solution, we could imagine something like that in the settings:
☑️ Enable platform tips
As a Collective admin, you can decide whether you prefer to be charged a fixed 5% platform fee or if you'd rather let your contributors add an optional tip on top of their contributions.
(note: the 5% part needs to be reviewed and discussed)

**Comment by alanna** [comment]
Pricing stuff is complex and difficult to continually keep changing - I don't believe we should go changing it around without due consideration. It must be fair across all users, easy to understand, and properly thought through. So while good points are being brought up here, we need to slow down and let OC take this on as a proper project before making changes.

**Comment by gusaus** [comment]
I totally agree and understand. That's why I'm trying to steer this to a solution proposed by OC and where someone @xdamman might know has also weighed in  https://github.com/opencollective/opencollective/issues/6377#issuecomment-1441898808

**Comment by gusaus** [comment]
Cross referencing what could be a more flexible solution for collectives using their website and/or third party platforms for fundraising https://github.com/opencollective/opencollective/issues/6377#issuecomment-1445476463

Considering the solution is already available for users of the open source platform powering over 40% of the internet, this should make Open Collective a very attractive option to a quite large [ecosystem/economy](https://wpengine.com/wordpress-economy/) as a fiscal sponsorship service, payment gateway, and financial management tool.

While [comparatively smaller](https://www.drupal.org/blog/who-sponsors-drupal-development-2020-2021-edition) than WordPress, Drupal and its nonprofit [Drupal Association](https://www.drupal.org/association) as an [early adopter](https://www.drupal.org/project/funding/issues/3344138) and [supporter](https://www.drupal.org/association/blog/a-more-sustainable-framework-for-fiscal-sponsorship-of-drupal-camps) will finally translate into widespread adoption and more funds flowing to sustain the OC platform and team. Especially with [Open Collective/drupal.org integration](https://www.drupal.org/project/funding/issues/3344138) finally moving forward. 

As long as the fee structure remains inconsistent (i.e. most 'off the OC platform' use cases will 'not' include a tip), a funding and discovery tool like https://discover.opencollective.com/ will be underutilized by those unable to use for most use cases. 

Reading back through these related issues, it seems like several key members of the OC platform team are open to the idea of hiding the platform tip (for a fee). Considering the uptick in revenue that 'will' be flowing from integrations and outside platforms, why not direct some resources (funds and [community contributors](https://opencollective.com/community-contributor-group)) into removing the barriers to making the platform a goto fundraising tool?

**Comment by gusaus** [comment]
Are #6799 and #6377 related attempts to solve some of the problems identified in #6252 and here?
